crates/web5/src/credentials/mod.rs

```diff
@@ -1,6 +1,17 @@
+mod create;
+mod credential_subject;
+mod data_model_validation;
+mod decode;
+mod issuer;
+mod jose_kit;
+mod jwt_payload_vc;
 pub mod presentation_definition;
+mod sign;
 pub mod verifiable_credential_1_1;
 
+pub use credential_subject::CredentialSubject;
+pub use issuer::{Issuer, ObjectIssuer};
+
 #[derive(thiserror::Error, Debug, Clone, PartialEq)]
 pub enum CredentialError {
```

Comment:
let's do this, since it's not part of the public APID surface, it's an implementation detail which rolls into `Web5Error` (I want to add `Web5Error` to the APID)

```suggestion
pub(crate) enum CredentialError {
```